examples/sklearn_elasticnet_wine/train.py

```diff
@@ -28,9 +28,11 @@ def eval_metrics(actual, pred):
     warnings.filterwarnings("ignore")
     np.random.seed(40)
 
-    # Read the wine-quality csv file (make sure you're running this from the root of MLflow!)
-    wine_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), "wine-quality.csv")
-    data = pd.read_csv(wine_path)
+    # Read the wine-quality csv file from the URL
+    try:
+        data = pd.read_csv('http://archive.ics.uci.edu/ml/machine-learning-databases/wine-quality/winequality-red.csv', sep=';')
+    except Exception as e:
```

Comment:
Could we instead use the Python `logging` module to still log the exception thrown by `read_csv`, e.g. as recommended [in this StackOverflow post](https://stackoverflow.com/a/31444861/12111347)?

Reply:
Is this correct?
```
try:
     data = pd.read_csv('http://archive.ics.uci.edu/ml/machine-learning-databases/wine-quality/winequality-red.csv', sep=';')
    
except Exception as e:
        logger.exception("Unable to download training & test CSV, check your internet connection. Error: %s", e)
```